ci/lattice_estimator.sage

```diff
@@ -36,7 +36,7 @@ def check_security(filename):
         try:
             # The lattice estimator is not able to manage such large dimension.
             # If we have the security for smaller `n` then we have security for larger ones.
-            if param.n >= 32768:
+            if param.n >= 16384:
```

Comment:
`>` is probably better suited here as we are fixing stuff if we are not inside the proper value range